skypeweb/skypeweb_messages.c

```diff
@@ -116,6 +116,22 @@ process_userpresence_resource(SkypeWebAccount *sa, JsonObject *resource)
 	// return FALSE;
 // }
 
+static void
+process_uri_message(const gchar* messagetype, SkypeWebAccount *sa, PurpleConversation* conv, const gchar* uri_content, time_t composetimestamp, const gchar* from) {

```

Comment:
Generally its better to 'namespace' functions (eg with the `skypeweb_` prefix) to prevent conflicts when static loading the plugins in, which bitlbee can do (for reference, you can see the function name collision in bitlbee when the old yahoo plugin conflicted with the new funyahoo plugin, and we had to change some function names to compensate)

Reply:
Note that exactly for this reason function was made static so that symbol does not get exported. Doesn't hurt to namespace it too though (done in latest version).